internal/credentials/credentials.go

```diff
@@ -27,6 +27,7 @@ const (
 	Kubeconfig          = "kubeconfig"
 	kubeadmincreds      = "kubeadmincreds"
 	kubeAdminKey        = "password"
+	seedReconfiguration = "seedReconfiguration"
```

Comment:
Is there any convention for key names in secrets? I feel like camel case is uncommon. Maybe `seed-reconfiguration` or `manifest.json` since that's the actual file name?